src/notification/index.js

```diff
@@ -114,7 +114,7 @@ export const getNarrowFromNotificationData = (
 const getInitialNotification = async (): Promise<Notification | null> => {
```

Comment:
Not sure about the name of this function, android provides us with a `readInitialNotification` and iOS provides us with a `getInitialNotification` this function calls one of them based on the platform. Should I split this and write platform conditional inside `handleInitialNotification`?

Reply:
Good question.

I just tested your #4758 repro instructions on iOS (translated slightly to the iOS UI), and it reproduces there. So the `getInitialNotifications` is an accurate name for that function that `PushNotificationIOS` gives us, it seems.

I think probably the right thing would be:
* Rename this function of ours to `readInitialNotification`, because that describes the behavior we *want* it to have, and that on Android it now does have.
* File an issue as the iOS counterpart of #4758, and add a TODO comment here pointing at that issue.

Reply:
> * File an issue as the iOS counterpart of #4758,

I've just filed this, as #4763.